applications/comment/initialize/init_comment.go

```diff
@@ -0,0 +1,50 @@
+package initialize
```

Comment:
nit: I think we no need to create a package named `initialize` for each micro services.